governance.md

```diff
@@ -0,0 +1,137 @@
+
+The Council reserves the right to eject current Members, if they are deemed to be actively harmful to the project’s well-being, and attempts at communication and conflict resolution have failed. This requires the consensus of the remaining Members.
+
+### Council decision making
+
+If it becomes necessary for the Steering Council to produce a formal decision, then they will use a form of the Apache Foundation voting process. This is a formalized version of consensus, in which +1 votes indicate agreement, -1 votes are vetoes (and must be accompanied with a rationale, as above), and one can also vote fractionally (e.g. -0.5, +0.5) if one wishes to express an opinion without registering a full veto. These numeric votes are also often used informally as a way of getting a general sense of people's feelings on some issue, and should not normally be taken as formal votes. A formal vote only occurs if explicitly declared, and if this does occur then the vote should be held open for long enough to give all interested Council Members a chance to respond -- at least one week.
+
+In practice, we anticipate that for most Steering Council decisions (e.g., voting in new members) a more informal process will suffice.
+
+### Conflict of interest
+
+It is expected that the Council Members will be employed at a wide range of companies, universities and non-profit organizations. Because of this, it is possible that Members will have conflict of interests. Such conflict of interests include, but are not limited to:
+
+* Financial interests, such as investments, employment or contracting work, outside of The Project that may influence their work on The Project.
+* Access to proprietary information of their employer that could potentially leak into their work with the Project.
+
+All members of the Council shall disclose to the rest of the Council any conflict of interest they may have. Members with a conflict of interest in a particular issue may participate in Council discussions on that issue, but must recuse themselves from voting on the issue.
+
+### Private communications of the Council
+
+To the maximum extent possible, Council discussions and activities will be public and done in collaboration and discussion with the Project Contributors and Community. The Council will have a private mailing list that will be used sparingly and only when a specific matter requires privacy. When private communications and decisions are needed, the Council will do its best to summarize those to the Community after eliding personal/private/sensitive information that should not be posted to the public internet.
+
+# Institutional partners and funding
```

Comment:
This section feels a bit heavy for Rasterio. 
I think it's still important to mention that we welcome institutions who want to participate but mostly (IMO) by sponsoring one or multiple of their own employees to contribute to rasterio.

Reply:
It struck me as being excessive for the current state of rasterio, too. We don't have plans for t-shirts or posters or conferences.

But mentioning Planet (for example, my employer) as an institutional partner is good for Planet, which wants more recognition for its open standards and open source initiatives, and is good for the project, I think. Same would go for Corteva, DevSeed, and a couple others which may qualify.

What in particular would you want to remove @vincentsarago ?